actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -548,6 +548,8 @@ def current_page?(options)
         request_uri = url_string.index("?") ? request.fullpath : request.path
         request_uri = URI.parser.unescape(request_uri).force_encoding(Encoding::BINARY)
 
+        url_string.sub!(%r{^/(.*)/$},'/\1')
```

Comment:
Ok, good to know.  Thank you for testing! :smile:  For what it's worth, using chomp with conditions is roughly 3x faster:
```
Comparison:
  chomp!:   1254105.3 i/s
    sub!:    400263.4 i/s - 3.13x slower
```